### ISSUE TITLE
I can’t setup this plugin correctly :(

### ISSUE THREAD

**Issue by skilastic** [plan-player-analytics/Plan#1249]
First of all, this plugin is great. But i have no clue on how to use it. I am using a nukkit server, and i dropped the file in my plugins folder. In the config, i can see that i have to put in a domain and external domain. I looked at the wiki explaining it, but i still don’t get it. How would i setup the page, and how would i know what to put in there? Also the plugin supports MySQL. I tried setting it up in the config, but it won’t connect. I would appreciate any help i can get, Because there is not a lot of tutorials out there. I prefer to watch a video instead of reading something i don’t understand. Anyways any help will be greatly appreciated :)

**Comment by Rsl1122** [comment]
Hi, based on your text I'm going to assume you have Plan running on your server already, with MySQL failing (so the plugin fails to enable)

### Connecting to mysql

Set `Database.Type` to MySQL and other MySQL settings to appropriate values.

Make sure that `Database.MySQL.Database` is a database that exists or one you have created in your MySQL with
```
CREATE DATABASE _dbname_
```
----

### Connecting to the Webserver on the plugin

First off see if you can access the page Plan gives in the enable log
```
Webserver Enabled on PORT #### (<Address is here>)
```
If not, check that the port is open in your firewall by using this tool https://www.yougetsignal.com/tools/open-ports/ - Put in your server IP and the port plan gave in the previous step.
If yes, nice, plugin is set up.

If the port is open but you still can't open the page, change `Webserver.Alternative_IP` settings to point to your server machine (IP or play.whatever.com) - make sure not to remove %port%

----

If you get stuck somewhere, you can join the discord to get more help: http://support.playeranalytics.net

**Comment by skilastic** [comment]
Sorry for my late reply, So far the plugin doesn’t seem to get along with luckperms. I checked if my port was open with the website you sent me, but it said it was closed. At the time of checking the port, I did not have the plugin on my server. I don’t think my server host would allow to open the port for me to use the website. Anyways, Thank you for the help, I hope there is a solution on my side :)
